core-java-modules/core-java-jndi/src/test/java/com/baeldung/jndi/JndiUnitTest.java

```diff
@@ -0,0 +1,60 @@
+package com.baeldung.jndi;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.springframework.jdbc.datasource.DriverManagerDataSource;
+import org.springframework.jndi.JndiTemplate;
+import org.springframework.mock.jndi.SimpleNamingContextBuilder;
+
+import javax.naming.*;
+import javax.sql.DataSource;
+
+import java.sql.SQLException;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class JndiUnitTest {
```

Comment:
This is a `JUnit 5` test, so we don't expect the class or methods to be `public` - just package private.